app/build.gradle

```diff
@@ -61,6 +61,7 @@ dependencies {
       'androidx.constraintlayout:constraintlayout:1.1.3',
       'androidx.core:core-ktx:1.0.2',
       'androidx.lifecycle:lifecycle-runtime-ktx:2.2.0-alpha03',
+      'androidx.recyclerview:recyclerview:1.0.0',
       'com.google.dagger:dagger:2.24',
       "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version",
```

Comment:
Please change double quote to single quotes

Reply:
Actually, I am not making changes to code which someone else has written.

Reply:
Last time in our discussions you suggested to fix issues if it comes in PR